src/auth_storage.js

```diff
@@ -0,0 +1,98 @@
+export class LocalStorage {
+  constructor (prefix = 'cozy:oauth:', getLocalStorage = () => window.localStorage) {
```

Comment:
I don't see why you pass a function as last parameter, instead of just the selected storage